internal/cmd/egctl/testdata/translate/in/backend-endpoint.yaml

```diff
@@ -57,3 +57,21 @@ spec:
   validation:
     wellKnownCACertificates: "System"
     hostname: www.example.com
+---
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: EnvoyExtensionPolicy
+metadata:
+  name: ext-proc-example
+spec:
+  targetRefs:
+    - group: gateway.networking.k8s.io
+      kind: HTTPRoute
+      name: myapp
```

Comment:
can we rename this to `backend` which is a valid HTTPRoute, so you'll get a valid `status` in o/p

Reply:
done